forge-gui-mobile/src/forge/itemmanager/views/ImageView.java

```diff
@@ -869,33 +869,18 @@ public void draw(Graphics g) {
                 if (isCollapsed) {
                     return;
                 }
-
-                float visibleLeft = getScrollLeft();
-                float visibleRight = visibleLeft + getWidth();
-                for (Pile pile : piles) {
-                    if (pile.getRight() < visibleLeft) {
-                        continue;
-                    }
-                    if (pile.getLeft() >= visibleRight) {
-                        break;
-                    }
-                    pile.draw(g);
-                }
-                return;
             }
 
-            final float visibleTop = getScrollValue();
-            final float visibleBottom = visibleTop + getScroller().getHeight();
-            for (ItemInfo itemInfo : items) {
-                if (itemInfo == null)
-                    continue;
-                if (itemInfo.getBottom() < visibleTop) {
+            float visibleLeft = getScrollLeft();
+            float visibleRight = visibleLeft + getWidth();
+            for (Pile pile : piles) {
```

Comment:
Do these piles exist if groupBy is null?

Reply:
Yes, they do. I tested it with combinations of groups and piles and they all work in manual testing.